docs/CucumberSyntax.md

```diff
@@ -22,11 +22,11 @@ More examples can be seen in the [generator cucumber features](https://github.co
 The framework supports setting configuration settings for the generator, defining the profile and describing the expected outcome. All of these are described below, all variable elements (e.g. `{generationStrategy}` are case insensitive), all fields and values **are case sensitive**.
 
 ### Configuration options
-* _the generation strategy is `{generationStrategy}`_ see [generation strategies](https://github.com/ScottLogic/datahelix/blob/master/docs/Options/GenerateOptions.md) - default: `full`
-* _the combination strategy is `{combinationStrategy}`_ see [combination strategies](https://github.com/ScottLogic/datahelix/blob/master/docs/Options/GenerateOptions.md) - default: `pinning`
-* _the walker type is `{walkerType}`_ see [walker types](https://github.com/ScottLogic/datahelix/blob/master/docs/Options/GenerateOptions.md) - default: `cartesian_product`
+* _the generation strategy is `{generationStrategy}`_ see [generation strategies](https://github.com/ScottLogic/datahelix/blob/master/generator/docs/GenerationTypes.md) - default: `random`
+* _the combination strategy is `{combinationStrategy}`_ see [combination strategies](https://github.com/ScottLogic/datahelix/blob/master/generator/docs/CombinationStrategies.md) - default: `pinning`
```

Comment:
Default is actually 'exhaustive' due to the change to use the `reductive` walker by default as part of #423